src/snyk/common/views/issueTreeProvider.ts

```diff
@@ -275,7 +294,10 @@ export abstract class ProductIssueTreeProvider<T> extends AnalysisTreeNodeProvid
   }
 
   protected getIssueFoundText(nIssues: number, _: number): string {
-    return `Snyk found ${!nIssues ? 'no issues! ✅' : `${nIssues} issue${nIssues === 1 ? '' : 's'}`}`;
+    if (!nIssues) {
+      return '✅ Congrats! No issues found!';
+    }
+    return `Snyk found ${nIssues} issue${nIssues === 1 ? '' : 's'}`;
```

Comment:
Small refactor for consistency. Agreed with Arvrid and Neil 🙌 

|Before|After|
|-|-|
|<img width="550" alt="before" src="https://github.com/user-attachments/assets/49b9f1fc-ea99-4f48-989e-a3ea800e6d3f">|<img width="541" alt="after" src="https://github.com/user-attachments/assets/56f29de3-4562-442e-a50f-b07b369b8075">|